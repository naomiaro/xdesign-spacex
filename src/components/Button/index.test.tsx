import React from 'react';
import ReactDOM from 'react-dom';
import { Button, BulletButton } from './index';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Button />, div);
});

describe('Rendering', () => {
  it('Renders a Button', () => {
    const wrapper = shallow(<Button>Refresh</Button>);
    expect(wrapper).toMatchSnapshot();
  });

  it('Renders a BulletButton', () => {
    const wrapper = shallow(<BulletButton>Refresh</BulletButton>);
    expect(wrapper).toMatchSnapshot();
  });
});
