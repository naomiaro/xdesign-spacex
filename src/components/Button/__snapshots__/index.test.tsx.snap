// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Renders a BulletButton 1`] = `
<Button
  className="bullet"
>
  Refresh
</Button>
`;

exports[`Rendering Renders a Button 1`] = `
<button
  className="btn btn-primary"
  type="button"
>
  Refresh
</button>
`;
